templates/index

```diff
@@ -279,6 +279,7 @@ cisco_xr_admin_show_environment_fan.textfsm, .*, cisco_xr, adm[[in]] sh[[ow]] en
 cisco_xr_show_cdp_neighbors_detail.textfsm, .*, cisco_xr, sh[[ow]] c[[dp]] neig[[hbors]] det[[ail]]
 cisco_xr_show_controllers_all_phy.textfsm, .*, cisco_xr, sh[[ow]] contr[[ollers]] (\S+) ph[[y]]
 cisco_xr_show_cef_drops_location.textfsm, .*, cisco_xr, sh[[ow]] cef drops loc[[ation]]
+cisco_xr_show_interfaces_summary.textfsm, .*, cisco_xr,sh[[ow]] int[[erface]] summ[[ary]]
```

Comment:
Put a space between the `cisco_xr,sh[[ow]]` and then add an **s** to the end of `int[[erfaces]]`

`cisco_xr_show_interfaces_summary.textfsm, .*, cisco_xr ,sh[[ow]] int[[erfaces]] summ[[ary]]`